app/jobs/spotlight/reindex_job.rb

```diff
@@ -7,7 +7,11 @@ class ReindexJob < Spotlight::ApplicationJob
     queue_as :default
 
     include Spotlight::JobTracking
-    with_job_tracking(resource: ->(job) { job.exhibit })
+    # Princeton Update
+    # We have tests that save resources that have no exhibit. If there's no
+    # exhibit, this would error.
```

Comment:
We don't have use cases for resources with no exhibit, though, right? Should we write a ticket to update the tests?

Reply:
I'm not sure - I was sort of assuming that the test exists for some reason, but I don't know what it would be. There's a bunch of edge cases sneaking around this code base, I just didn't want to accidentally reintroduce an old bug.